libsyclinterface/dbg_build.sh

```diff
@@ -2,30 +2,31 @@
 set +xe
 rm -rf build
 mkdir build
-pushd build
+pushd build || exit 1
 
-INSTALL_PREFIX=`pwd`/../install
+INSTALL_PREFIX=$(pwd)/../install
 rm -rf ${INSTALL_PREFIX}
 
 cmake                                                       \
     -DCMAKE_BUILD_TYPE=Debug                                \
     -DCMAKE_C_COMPILER=icx                                  \
-    -DCMAKE_CXX_COMPILER=dpcpp                              \
+    -DCMAKE_CXX_COMPILER=icpx                               \
+    -DCMAKE_CXX_FLAGS=-fsycl                                \
```

Comment:
Probably the right way of integrating SYCL and cmake would be to use `find_package(IntelSycl)`, but this could be done in a separate PR.

Reply:
These changes are in the helper script, let us fix in separate PR. I just needed to get the helper script working once more to develop on libsyclinterface.